CHANGES_NEXT_RELEASE

```diff
@@ -1,6 +1 @@
-- Fix: update tests to not create initial entity when append mode is enabled
-- Fix: update tests according new default value for appendMode
-- Fix: use 'options=upsert' when update ngsiv2 CB entities and appendMode is enabled (iotagent-node-lib#956)
-- Fix: do not propagate group config (timestamp and explicitAttrs) to autoprovisioned devices (at database level) (iotagent-node-lib#1377)
-- Fix: renamed iotagent-ul to iotagent-json in commandsPolling-test (#726)
-- Remove: RPM stuff (no longer used)
```

Comment:
Include this part in rpm/SPECS/iotajson.spec, as consolidated changelog

(Someday we will migrate that .spec to just Changelog, like in other repos :)

Reply:
rpm/SPECS/iotajson.spec was removed

Reply:
Use https://github.com/telefonicaid/iotagent-json/blob/master/Changelog

Reply:
fixed in https://github.com/telefonicaid/iotagent-json/pull/740/commits/45c850740b13a69ff60ec23de7215c58826ea337